game.js

```diff
@@ -547,69 +547,60 @@ module.exports = function createGame(options) {
         return lodash.intersection(state.roles, lodash.flatten([roles]))[0];
     }
 
-    function command(playerIdx, command) {
-        debug('command from player: ' + playerIdx);
-        debug(command);
-        var i, action, message;
-        var playerState = state.players[playerIdx];
-        if (playerState == null) {
-            throw new GameException('Unknown player');
+    const playerIsReady = (command, playerIdx) => {
```

Comment:
I would prefer to be consistent with the existing style, rather than have a mixture of styles:

    function playerIsReady(command, playerIds) {

Unless there is any reason for this syntax?

Reply:
Nope, no reason. Habit